packages/react-tooltip/Spec.md

```diff
@@ -1,63 +1,474 @@
+  componentRef?: React.Ref<TooltipManagerApi>;
+}
+
+export const tooltipManagerShorthandProps = ['tooltip'] as const;
+
+export type TooltipManagerState = ComponentState<
+  TooltipManagerProps & {
+    /**
+     * The Tooltip being rendered, if any.
+     */
+    tooltip?: ShorthandProps<TooltipProps>;
+  }
+>;
+```
+
+# Structure
+
+## Public
+
+```tsx
+<TooltipProvider>
+  <Button tooltip="Hello world" /> {/* for example... */}
+</TooltipProvider>
+```
+
+## Internal
+
+`TooltipProvider`:
+
+```tsx
+<internal__TooltipProviderContext.Provider value={{ current: undefined }}>
+  <slots.root {...slotProps.root} />
+  <React.Suspense fallback={null}>
+    <TooltipManager />
+  </React.Suspense>
+</internal__TooltipProviderContext.Provider>
+```
+
+`TooltipManager`:
+
+```tsx
+<slots.root {...slotProps.root}>
+  {state.children}
+  <slots.tooltip {...slotProps.tooltip} />
+</slots.root>
+```
+
+`Tooltip`:
+
+```tsx
+<slots.root {...slotProps.root}>
+  <slots.arrow {...slotProps.arrow} />
+  {state.children}
+</slots.root>
+```
+
+## DOM
+
+How the component will be rendered as HTML elements
 
-- _Link to Open UI research_
-- _Link to comparison of v7 and v0_
-- _Link to GitHub epic issue for the converged component_
+```tsx
+<div {/* TooltipProvider */}>
+  <button aria-describedby="tooltip42" /> {/* for example... */}
 
-## Sample Code
+  <div {/* TooltipManager */}>
+    <div id="tooltip42" {/* Tooltip */}>
+      <div {/* Arrow */} />
+      Hello world
+    </div>
+  </div>
+</div>
+```
 
-_Provide some representative example code that uses the proposed API for the component_
+# Migration
 
-## Variants
+## Migration from v8
 
-_Describe visual or functional variants of this control, if applicable. For example, a slider could have a 2D variant._
+The converged API does not support many of the custom features of the v8 tooltip. We may need to revisit and add additional features to the converged Tooltip if needed.
 
-## API
+- `Tooltip`
+  - `calloutProps` => Not supported. The arrow's visibility can be controlled using `noArrow`.
+  - `componentRef` => Not supported. The tooltip can't be manually invoked.
+  - `content="..."` => Either a component's `tooltip="..."` prop or the Tooltip's `children` prop.
+  - `delay` => `showDelay`
+  - `directionalHint` => `placement`
+    - `DirectionalHint.topLeftEdge` => `placement="top-start"`
+    - `DirectionalHint.topCenter` => `placement="top"`
+    - `DirectionalHint.topRightEdge` => `placement="top-end"`
+    - `DirectionalHint.topAutoEdge` => Not supported
+    - `DirectionalHint.bottomLeftEdge` => `placement="bottom-start"`
+    - `DirectionalHint.bottomCenter` => `placement="bottom"`
+    - `DirectionalHint.bottomRightEdge` => `placement="bottom-end"`
+    - `DirectionalHint.bottomAutoEdge` => Not supported
+    - `DirectionalHint.leftTopEdge` => `placement="left-start"`
+    - `DirectionalHint.leftCenter` => `placement="left"`
+    - `DirectionalHint.leftBottomEdge` => `placement="left-end"`
```

Comment:
I drew up a positioning table for v8 and v0 in the `Menu` spec: https://github.com/microsoft/fluentui/blob/master/packages/react-menu/Spec.md#positionalignment-hints

What do you think about omitting left/right so that syntax is independent of LTR/RTL ?